source/quickstart/go/gorilla.md

```diff
@@ -48,13 +48,13 @@ Push to the master branch of the Aptible Git remote:
 
     git push aptible master
 
-If your app deploys successfully, a message will appear near the end of the remote output with a default VHOST:
+If your app deploys successfully, a message will appear near the end of the remote output with a default SSL/TLS endpoint:
```

Comment:
Is this still the case? I thought you had to configure those manually now. If so we probably need to update all our quickstart guides.

cc @fancyremarker